frontend/public/components/_secret.scss

```diff
@@ -0,0 +1,31 @@
+.co-create-secret-form {
+  .btn-file {
```

Comment:
The file picker is generic and will eventually be used outside the secret form. We shouldn't scope the styles to the secret form.

Please use `co-` prefix for new styles.